api/src/test/java/org/apache/iceberg/transforms/TestProjection.java

```diff
@@ -90,6 +91,28 @@ public void testIdentityProjection() {
     }
   }
 
+  @Test
+  public void testTimestampNanosIdentityProjection() {
+    org.apache.iceberg.Schema schema =
+        new org.apache.iceberg.Schema(
```

Comment:
The package `org.apache.iceberg` looks redundant as this class already imports `Schema`.